desc/magnetic_fields.py

```diff
@@ -1803,6 +1810,299 @@ def from_surface(
         )
 
 
+class OmnigenousField(Optimizable, IOAble):
+    """Model field with perfect omnigenity.
+
```

Comment:
need a more descriptive docstring for this, describe the keywords and args of the init method

Reply:
Updated. Let me know if it is detailed enough

Reply:
Can you add in the description some of the math? like saying `L_B` is for radial chebyshev polynomials is not very descriptive. And referencing to your paper and saying which variables match what in the paper would be good too